ena-submission/src/ena_deposition/ena_submission_helper.py

```diff
@@ -354,24 +352,29 @@ def get_authors(authors: str) -> str:
     return authors
 
 
-def get_country(metadata: dict[str, str]) -> str:
-    country = metadata.get("geoLocCountry", "Unknown")
-    admin_levels = ["geoLocAdmin1", "geoLocAdmin2", "geoLocCity", "geoLocSite"]
-    admin_values = [val for level in admin_levels if (val := metadata.get(level))]
-    admin = ", ".join(admin_values)
-    return f"{country}: {admin}" if admin else country
+def _gzip_string(content: str) -> bytes:
+    buffer = io.BytesIO()
+    with gzip.GzipFile(fileobj=buffer, mode="w") as gz_file, \
+        io.TextIOWrapper(gz_file, encoding="utf-8") as wrapper:
+        wrapper.write(content)
+    return buffer.getvalue()
 
 
-def create_flatfile(
-    config: Config, metadata, organism_metadata, unaligned_nucleotide_sequences, dir
+def download_flatfile(
+    config: Config,
+    metadata,
+    dir
 ):
-    collection_date = metadata.get("sampleCollectionDate", "Unknown")
-    country = get_country(metadata)
-    organism = organism_metadata.get("scientific_name", "Unknown")
     accession = metadata["accession"]
-    description = get_description(config, metadata)
-    authors = get_authors(metadata.get("authors", ""))
-    moleculetype = get_molecule_type(organism_metadata)
+    version = metadata["version"]
+    category = "annotations"
+    filename = f"{accession}.embl"
```

Comment:
I think it would be good to rename the file as 
```suggestion
    filename = f"{accession}.{version}.embl"
```